etc/get_dep_version.py

```diff
@@ -0,0 +1,21 @@
+import json
+import sys
+
+# Prints out the name of a dependency as specified in Package.resolved.
+# Usage: python3 get_dep_version.py [package name]
+# for example: python3 get_dep_version.py swift-bson
+
+if len(sys.argv[1:]) != 1:
+    print("Error: Expected exactly one argument to program, got: {}".format(sys.argv[1:]))
+    exit(1)
```

Comment:
(I'm now realizing these exit(1)s don't prevent the script from continuing onward on failure, will push up a fix for this shortly)